resources/services/cosmosdb/sql_databases.go

```diff
@@ -0,0 +1,142 @@
+package cosmosdb
+
+import (
+	"context"
+
+	"github.com/cloudquery/cq-provider-azure/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func CosmosDBSqlDatabases() *schema.Table {
+	return &schema.Table{
+		Name:         "azure_cosmosdb_sql_databases",
+		Description:  "Azure Cosmos DB SQL database.",
+		Resolver:     fetchCosmosdbSqlDatabases,
+		Multiplex:    client.SubscriptionMultiplex,
+		DeleteFilter: client.DeleteSubscriptionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"subscription_id", "id"}},
+		Columns: []schema.Column{
+			{
+				Name:        "subscription_id",
+				Description: "Azure subscription id",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAzureSubscription,
+			},
+			{
+				Name:        "database_id",
```

Comment:
If it duplicates `id` column  it might be redundant, and other columns like this